tgui/packages/tgui/interfaces/Airlock.js

```diff
@@ -0,0 +1,110 @@
+import { useBackend } from '../backend';
+import { Box, Button, Flex, LabeledList, Section, Divider } from '../components';
+import { Window } from '../layouts';
+
+export const Airlock = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    signalers,
+    wireColors,
+    wireStates,
+    netId,
+    powerIsOn,
+    boltsAreUp,
+    aiControlDisabled,
+    safety,
+  } = data;
+
+  const handleWireInteract = (wireColorIndex, action) => {
+    act(action, { wireColorIndex });
+  };
+
+  const wires = Object.keys(wireColors);
+
+  return (
+    <Window
+      height={460}
+      width={370}>
+      <Window.Content>
+        <Section
+          title="Access panel">
+          <Box>
+            {"An identifier is engraved under the airlock's card sensors:"} <Box inline italic>{netId}</Box>
+          </Box>
+          <Divider />
+          <LabeledList>
+            { wires.map((entry, i) => (
+              <LabeledList.Item
+                key={entry}
+                label={(entry + " wire")}
+                labelColor={entry.toLowerCase()}>
+                {
+                  !wireStates[i]
+                    ? (
+                      <Box
+                        height={1.8} >
+                        <Button
+                          icon="cut"
+                          content="Cut"
+                          onClick={() => handleWireInteract(i, "cut")} />
+                        <Button
+                          icon="bolt"
+                          content={"Pulse"}
+                          onClick={() => handleWireInteract(i, "pulse")} />
+                        <Button
+                          icon="broadcast-tower"
+                          width={10}
```

Comment:
Explicit width set to look neat based on label size difference between "Attach Signaler" and "Detach Signaler"? If so, based on screenshots it could probably be `11`.

Reply:
`11` was a bit much but `10.5` seems okay.